packages/alfa-act/src/question.ts

```diff
@@ -9,38 +9,49 @@ import * as json from "@siteimprove/alfa-json";
 /**
  * @public
  */
-export class Question<Q, S, A, T = A>
+export class Question<Q, S, C, A, T = A>
   implements
     Functor<T>,
     Applicative<T>,
     Monad<T>,
-    Serializable<Question.JSON<Q, S>>
+    Serializable<Question.JSON<Q, S, C>>
 {
-  public static of<Q, A, S>(
+  public static of<Q, S, C, A>(
     uri: string,
     type: Q,
     subject: S,
+    context: C,
```

Comment:
Same here, main use case seems to be asking questions whose subject is the same as the context. Defaulting (or overloading) would make call sites cleaner.